### ISSUE TITLE
Select Glyphs With Errors

### ISSUE THREAD

**Issue by weiweihuanghuang** [jenskutilek/RedArrow#1]
```
Deprecated: use 'buildAdditionContextualMenuItems'.
Traceback (most recent call last):
  File "Select Glyphs With Errors.py", line 31, in <module>
  File "Select Glyphs With Errors.py", line 20, in run_test
  File "lib/fontObjects/robofabWrapper.pyc", line 2797, in draw
  File "/Applications/RoboFont.app/Contents/Resources/lib/python2.7/defcon/objects/glyph.py", line 299, in draw
  File "/Applications/RoboFont.app/Contents/Resources/lib/python2.7/defcon/objects/glyph.py", line 306, in drawPoints
  File "/Applications/RoboFont.app/Contents/Resources/lib/python2.7/defcon/objects/contour.py", line 434, in drawPoints
  File "/Applications/RoboFont.app/Contents/Resources/lib/python2.7/robofab/pens/pointPen.py", line 113, in endPath
  File "/Applications/RoboFont.app/Contents/Resources/lib/python2.7/robofab/pens/adapterPens.py", line 88, in _flushContour
AttributeError: OutlineTestPen instance has no attribute 'moveTo'
```

**Comment by weiweihuanghuang** [comment]
Actually still get this error

**Comment by jenskutilek** [comment]
Thanks! Fixed in 466c296d586a233d1514315c361f08559b7e180b.
